Microsoft.Toolkit.Uwp.UI.Controls.Core/DropShadowPanel/IAlphaMaskProvider.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Composition;
+
+/// <summary>
+/// Any user control can implement this interface to provide a custom alpha mask to it's parent <see cref="DropShadowPanel"/>
+/// </summary>
+public interface IAlphaMaskProvider
```

Comment:
Can we move this to the `Microsoft.Toolkit.Uwp.UI` library?

If no namespace is provided where does it end up, the root of the assembly's one?